wp-cli-rename-db-prefix.php

```diff
@@ -213,21 +213,28 @@ protected function update_blog_options_tables() {
 		// todo should this really go after update_options_table, and reuse the same query?
 		// todo is this running on the root site twice b/c update_options_table() hits that too? should call either that or this, based on is_multisite() ?
 
-    	$sites = wp_get_sites( array( 'limit' => false ) );   //todo can't use b/c already renamed tables?
+		//Check Wordpress Version > 4.6
+		if ( function_exists( 'wp_get_sites' ) ) {
+			$sites = wp_get_sites( array( 'limit' => false ) );
+		} else {
+			$sites = get_sites( array( 'number' => false ) );
+		}
 		//blogs = $wpdb->get_col( "SELECT blog_id FROM `" . $this->new_prefix . "blogs` WHERE public = '1' AND archived = '0' AND mature = '0' AND spam = '0' ORDER BY blog_id DESC" );
 
 		if ( ! $sites ) {
 			throw new Exception( 'Failed to get all sites.' );  // todo test
 		}
 
 		foreach ( $sites as $site ) {
+			$object_vars  = get_object_vars( $site );
+			$blog_id      = $object_vars["blog_id"];
 			$update_query = $wpdb->prepare( "
-				UPDATE `{$this->new_prefix}{$site->blog_id}_options`
+				UPDATE `{$this->new_prefix}{$blog_id}_options`
 				SET   option_name = %s
 				WHERE option_name = %s
 				LIMIT 1;",
-				$this->new_prefix . $site->blog_id . '_user_roles',
-				$this->old_prefix . $site->blog_id . '_user_roles'
+				$this->new_prefix . $blog_id . '_user_roles',
+				$this->old_prefix . $blog_id . '_user_roles'
 			);
```

Comment:
If we switch to only using `get_sites()` then I don't think these changes will be necessary.